src/App/App.js

```diff
@@ -1,46 +1,55 @@
-import * as React from "react";
-import { useState, useCallback } from "react";
+import * as React from 'react';
+import { useState, useCallback } from 'react';
 
 import './styles.scss';
 
 function App() {
   const [tiles, setTiles] = useState([]);
   const [shuffledTiles, setShuffledTiles] = useState([]);
   const [isTilesShuffled, setIsTilesShuffled] = useState(false);
-  const orderTiles = useCallback((async) => {
-    const order = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-    let tilesInProgress = [...tiles];
-    if (tiles.length === 0) {
-      for (let i = 0; i < order.length; i++) {
+  const orderTiles = useCallback(
+    (async) => {
+      const order = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+      let tilesInProgress = [...tiles];
+      if (tiles.length === 0) {
+        for (let i = 0; i < order.length; i++) {
+          tilesInProgress.push(
+            <div className="drag-item" draggable="true" key={i}>
+              <p className="tile-number">{order[i]}</p>
+            </div>,
+          );
+        }
         tilesInProgress.push(
-          <div className="drag-item" draggable="true" key={i}>
-            <p className="tile-number">{order[i]}</p>
-          </div>
-        )
+          <div
+            className="drag-item"
+            draggable="true"
+            id="empty"
+            key="empty"
+          ></div>,
```

Comment:
I think that you can just do /> if the div is empty?

```` 
    <div
            className="drag-item"
            draggable="true"
            id="empty"
            key="empty"
          />

````

Reply:
done